packages/node_modules/@webex/webex-core/src/lib/services/service-catalog.js

```diff
@@ -335,7 +335,9 @@ const ServiceCatalog = AmpState.extend({
 
       this.once(serviceGroup, () => resolve());
```

Comment:
We should [clearTimeout](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/clearTimeout) once resolved, just in case we get to a race condition where  the promise is attempting to resolve and reject at the same time.